### PR TITLE
Add vanilla CSS support to HoverCard 

### PR DESCRIPTION
**Pull request by andy-hook**

### CODE REVIEW

File: components/demos/HoverCard/stitches/index.jsx [hunk 1]
```diff
@@ -33,38 +77,17 @@ const StyledContent = styled(HoverCardPrimitive.Content, {
...
-    '&[data-state="open"]': {
-      '&[data-side="top"]': { animationName: slideDownAndFade },
-      '&[data-side="right"]': { animationName: slideLeftAndFade },
-      '&[data-side="bottom"]': { animationName: slideUpAndFade },
-      '&[data-side="left"]': { animationName: slideRightAndFade },
-    },
+    '&[data-side="top"]': { animationName: slideDownAndFade },
+    '&[data-side="right"]': { animationName: slideLeftAndFade },
+    '&[data-side="bottom"]': { animationName: slideUpAndFade },
+    '&[data-side="left"]': { animationName: slideRightAndFade },
```
Comment (benoitgrelard) [inline components/demos/HoverCard/stitches/index.jsx:83]: Why this change?
The recommendation should be to qualify it right?

**Review by benoitgrelard — APPROVED** [review]

File: components/demos/HoverCard/stitches/index.jsx (hunk 1 above)
Comment (andy-hook) [reply components/demos/HoverCard/stitches/index.jsx:83]: Yes you're right, I must have tripped. Presumably the examples should match?

e.g. https://www.radix-ui.com/docs/primitives/components/dropdown-menu#collision-aware-animations

Comment (benoitgrelard) [reply components/demos/HoverCard/stitches/index.jsx:83]: Yeah I'll add an item to the last pass checklist to go through for consistency.

Comment (andy-hook) [reply components/demos/HoverCard/stitches/index.jsx:83]: > not sure if you saw but I added a last pass checklist

I didn't but this is good stuff, was going to suggest similar. I'll revert this particular change and we'll address others there.

**Review by benoitgrelard — APPROVED** [review]
